azext_edge/tests/edge/init/test_work_unit.py

```diff
@@ -387,7 +387,7 @@ def _get_resources_of_type(resource_type: str, template: TemplateVer):
             None,  # cluster_namespace
             generate_random_string(),  # resource_group_name
             None,  # keyvault_resource_id
-            None,  # keyvault_sat_secret_name
+            None,  # keyvault_spc_secret_name
```

Comment:
personally prefer dictionaries to avoid worrying about order/making sure enough params are in each scenario but if you decide to do changes, it can be done in a later pr

Reply:
Too much scope for this PR.